sensor_msgs/PointCloud[] clusters
tabletop_object_detector/Table table
---

string label
int32 model_id
nrg_object_recognition/pose pose

